Asuka broken on Beta / Master

```
../../.pub-cache/hosted/pub.dev/asuka-2.0.2/lib/src/asuka_singleton.dart:345:29: Error: The argument type 'bool?' can't be assigned to the parameter type 'bool' because 'bool?' is nullable and 'bool' isn't.
      semanticsDismissible: semanticsDismissible,
                            ^
Target kernel_snapshot failed: Exception

Command PhaseScriptExecution failed with a nonzero exit code
** BUILD FAILED **
```

--- a/lib/src/asuka_singleton.dart
+++ b/lib/src/asuka_singleton.dart
@@ -332,7 +332,7 @@ class _Asuka {
     Color barrierColor = cupertino.kCupertinoModalBarrierColor,
     bool barrierDismissible = true,
     bool useRootNavigator = true,
-    bool? semanticsDismissible,
+    bool semanticsDismissible = false,
     RouteSettings? routeSettings,
   }) {
     return cupertino.showCupertinoModalPopup<T>(

